webapp-django/crashstats/crashstats/utils.py

```diff
@@ -54,6 +54,11 @@ def wrapper(request, *args, **kw):
         response = f(request, *args, **kw)
 
         if isinstance(response, http.HttpResponse):
+            if (
+                isinstance(response, http.HttpResponseBadRequest) and
+                not response['Content-Type'].startswith('application/json')
+            ):
+                response['Content-Type'] = 'application/json; charset=UTF-8'
```

Comment:
Note taken about the necessary comment. 
Also, I think you're right. We should ALWAYS set the content-type to `application/json` on views with this decorator. A common use-case is this:

```python
@json_view
def myview(request):
    if some_reason:
        return redirect(reverse('url'))
```
I think the `Location` header trumps the `Content-Type` header when the browser gets one of these. E.g.
```
HTTP 302 Moved
Content-Type: application/json; charset=UTF-8
Location: http://localhost:8000/other/url
```